CMakeLists.txt

```diff
@@ -58,6 +58,12 @@ endif ()
 include(RippledCompiler)
 include(RippledInterface)
 
+option(only_docs "Include only the docs target?" FALSE)
```

Comment:
Who's asking who?

Reply:
note - just a help string: if you use the cmake GUI, it will show all the options of the project. this will be a label on the option. most people won't see it.